bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -19,6 +22,9 @@ spec:
     metadata:
       labels: {{- include "postgresql-ha.labels" . | nindent 8 }}
         app.kubernetes.io/component: postgresql
+        {{- with .Values.postgresql.podLabels }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
Please, could you use `"postgresql-ha.tplValue"`? and have it as a template value too.